src/v3/src/transformer/selectAuthenticator/transformSelectAuthenticatorEnroll.ts

```diff
@@ -142,6 +148,22 @@ export const transformSelectAuthenticatorEnroll: IdxStepTransformer = ({
     },
   };
 
+  const customLink = getGracePeriodRequiredSoonCustomLink(widgetProps);
+  let gracePeriodRequiredSoonCustomLink: LinkElement | undefined;
+  if (customLink) {
+    const { href, text } = customLink;
+    gracePeriodRequiredSoonCustomLink = {
+      type: 'Link',
+      options: {
+        href,
+        target: '_blank',
+        step: '',
+        label: text,
+        dataSe: 'gracePeriodRequiredSoonCustomLink',
+      },
+    };
+  }
+
```

Comment:
Does this also enforce that both href and text have to be defined as mentioned in my comment above? Just want to confirm that v2 + v3 behavior are identical

Reply:
Updated this to be like v2 (needs both href and text to be defined) since API is enforcing that when customizing the link.